misc/assets/anonymization.svg

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
@martinmacko47 V akom editore si vytvaral diagram? Ja pouzivam [draw.io](https://www.draw.io/), a ked exportujem diagram ako `.svg`, tak ten subor ma ine formatovanie oproti tvojmu vzorovemu v `attachments.svg`.

Reply:
@viliambalaz Rucne v textovom editore. Akykolvek vizualny editor pouzijes, spravi to neprehladne diffy.

Reply:
:ok: